### PR TITLE
screenplay-integration-example start

### PR DESCRIPTION
**Pull request by Bnaya**

### CODE REVIEW

**Comment by benjamingr** [comment]
Feel free to commit into this repo liberally and without our due process (branch names, JIRAs etc) but note that this repo is public :]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+.screenplay
```
Comment (rannn505) [inline .gitignore:2]: maybe you want to add it to the documentation so users dont get angry :)

File: integration-examples/mocha/mocha-example-test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// const assert = require('chai').expect;
```
Comment (rannn505) [inline integration-examples/mocha/mocha-example-test.js:1]: remove this :)

File: integration-examples/mocha/mocha-example-test.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+const puppeteer = require('puppeteer');
+const { attach, updateHistoryFromScreenplayResultsOnly } = require('@testim/screenplay');
+describe("screenplay mocha example", () => {
```
Comment (rannn505) [inline integration-examples/mocha/mocha-example-test.js:4]: looks good and works good :)

File: integration-examples/mocha/mocha-example.test.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        await page.goto("https://google.com");
+        await page.click('#lbalblabal');
+    }).timeout(10000);
+
+    it('Yet Another test 3', async () => {
+        await page.goto("https://ynet.co.il");
+        await page.click('#lbalblabal');
+    }).timeout(10000);
+
+    it('Yet Another test 4 pass!!!', async () => {
```
Comment (benjamingr) [inline integration-examples/mocha/mocha-example.test.js:61]:
```suggestion
    it('Yet Another test 4 pass', async () => {
```

File: integration-examples/jest/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "jest",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@testim/screenplay": "^0.10.2",
```
Comment (eladrkatz) [inline integration-examples/jest/package.json:8]: any reason to lock this to a specific version ?

File: integration-examples/jest/package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    "@types/puppeteer": "^3.0.0",
+    "expect-puppeteer": "^4.4.0",
```
Comment (eladrkatz) [inline integration-examples/jest/package.json:10]: add license?

Comment (Bnaya) [reply integration-examples/jest/package.json:10]: It's a demo, and not published package,
But it does make sense to specify licence in repo level so users won't be afraid to copy paste our examples
@benjamingr MIT? BSD3 what should it be?

File: integration-examples/jest/package.json (hunk 5 above)
Comment (Bnaya) [reply integration-examples/jest/package.json:8]: Yes, and we also have lock file
Upgrades needs to be done explicitly

File: integration-examples/jest/package.json (hunk 6 above)
Comment (benjamingr) [reply integration-examples/jest/package.json:10]: MIT is fine

File: integration-examples/launch-api/package.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (eladrkatz) [inline integration-examples/launch-api/package.json:1]: i think `simple` might be a better folder name than launch-api

File: integration-examples/launch-api/tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,69 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```
Comment (eladrkatz) [inline integration-examples/launch-api/tsconfig.json:3]: maybe remove all commented lines?

File: integration-examples/launch-api/package.json [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+  "scripts": {
+    "run-example": "node example.js",
+    "run-example-typescript": "ts-node example-ts.ts",
+    "test": "npm run run-example && npm run run-example-typescript"
+  },
+  "private": true,
+  "devDependencies": {
+    "@testim/screenplay": "^0.10.2",
+    "@types/puppeteer": "^3.0.0",
+    "puppeteer": "^4.0.0",
```
Comment (eladrkatz) [inline integration-examples/launch-api/package.json:14]: @testim/screenplay@0.10.2" has incorrect peer dependency "puppeteer@^3.0.4".

File: integration-examples/launch-api/package.json (hunk 7 above)
Comment (benjamingr) [reply integration-examples/launch-api/package.json:1]: I think that makes sense now that `simple` (that used endTest) was removed.

File: integration-examples/launch-api/example.js [hunk 10]
```diff
@@ -0,0 +1,40 @@
+const screenplay = require('@testim/screenplay');
+
+screenplay.launch({ testName: __filename }, async function(page) {
```
Comment (eladrkatz) [inline integration-examples/launch-api/example.js:3]: launch API should probably wait for any key to close like `screenplay show` (I know this isn't related to this PR but still..)

File: integration-examples/mocha/mocha-example.test.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+        await updateHistoryFromScreenplayResultsOnly(runId);
+        await browser.close();
+    });
+
+    it('returns Chrome Puppeteer Github repo as first search result', async () => {
+        await page.goto("https://github.com");
+        await page.click('#some-element');
+    }).timeout(10000)
+
+    it('Yet Another test', async () => {
```
Comment (eladrkatz) [inline integration-examples/mocha/mocha-example.test.js:51]: naming of all of these tests should be revisited

File: integration-examples/launch-api/package.json (hunk 9 above)
Comment (Bnaya) [reply integration-examples/launch-api/package.json:14]: We do need to change `puppeteer` to a range in the package

**Pull request merged**